microsoft-edge/extensions/publish/marketing-guidelines.md

```diff
@@ -0,0 +1,121 @@
+ms.service: microsoft-edge
+ms.date: 08/13/2025
+---
+# Marketing guidelines for Microsoft Edge Add-ons
+<!-- todo: is this content copied (then modified) from a webpage? which? -->
+
+<!-- todo: global in this article: lexicon:
+the Microsoft Edge Add-on badge
+the Microsoft Edge add-on badge
+the Microsoft Edge add-ons badge
+the Microsoft Edge Add-ons badge
+the Featured badge
+-->
+
+<!-- todo: global in Extensions docs: which?
+add-ons (such as extensions, themes, or web apps)
+add-ons (such as extensions, themes, or apps)
+-->
+
+These guidelines outline how to use Microsoft Edge brand assets (such as badges and logos) across websites, documentation, tutorials, and other services or integrations.<!-- todo: ", especially the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) site"?  is this article specifically for the Microsoft Edge Add-ons site, or other sites as well? -->  They ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.
+
+Items at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) include extensions, themes, and web apps.
+
+See also:
+* [The Featured badge](../index.md#the-featured-badge) in _Overview of Microsoft Edge extensions_.
+
+
+<!-- ====================================================================== -->
+## The Microsoft Edge Add-on badge
+<!-- todo:
+is the "Microsoft Edge Add-on(s) badge" the same thing as "the Featured badge"?
+mention [The Featured badge](../index.md#the-featured-badge)? -->
+
+<!-- todo: .png -->
+
+A Microsoft Edge Add-on badge is a visual asset designed to help you promote Edge Add-ons and build user trust.
+
+The Microsoft Edge Add-on badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-on badge is intended exclusively for promoting add-ons (such as extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.
+
+Unlike generic Microsoft Store badges,<!-- todo: .png(s) --> the Microsoft Edge Add-on badge provides a distinct, trusted visual cue that connects your add-on directly to the Microsoft Edge ecosystem.
+<!-- todo: which?
+generic Microsoft Store badges - what's meant? show
+the Microsoft Store logo - show, if relevant here
+-->
+
+Use the provided badges in the `.zip` file (below), rather than creating your own image files.
+
+
+<!-- ====================================================================== -->
+## Minimum size for badges
+
+To ensure that the badge text remains readable, display it at a minimum size of 32px.  Avoid using smaller sizes.
+
+When scaling the badge, always maintain its aspect ratio; do not stretch or compress the badge image.  This helps preserve visual quality and reinforces your product's Credibility.<!-- todo: credibility? -->
```

Comment:
is Credibility a specialized tech term here, eg a metric?  else lowercase

@G-u-r-j-e-e-t